hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieMergeHandle.java

```diff
@@ -446,6 +446,8 @@ public void performMergeDataValidationCheck(WriteStatus writeStatus) {
       return;
     }
 
+    IOUtils.checkParquetFileVaid(hoodieTable.getHadoopConf(), newFilePath);
+
     long oldNumWrites = 0;
```

Comment:
In what senario the handle flushes successfully but the parquet is in incompleteness?